lib/api.js

```diff
@@ -16,13 +16,17 @@ try {
 const defaultPageSize = 100
 const defaultPostCount = 10
 
+function getErrorMessage(e) {
+  return e?.cause?.data?.detail || "Unexpected error occurred"
+}
+
 export async function getLandingPage(slug) {
   try {
     const page = await butter.page.retrieve('landing-page', slug);
 
     return page?.data?.data;
   } catch (e) {
-    throw e.response.data.detail
+    throw getErrorMessage(e)
```

Comment:
hate to do this to you, but can we change `e` to `err`? `e` is by convention meant for `event` instead of `error`. sorry!